_plays/08.md

```diff
@@ -5,11 +5,11 @@ title: Make it easy to use
 
 ### Checklist
 1.  Provide reference implementation.  Show how to use the API.
-2.  Write modifiable examples.
+2.  Write modifiable examples.  Take a look at this Mapbox [example](https://www.mapbox.com/mapbox.js/example/v1.0.0) for reference.
 3.  Even create client-side wrappers with use, if you have the resources.  This brings the service even closer to the end-use.
 4.  Try it out yourself!
 
 ### Key Questions
 - How long does it take to build a toy application on the service?
 - Can the service be used in a hackathon?  Even if you are not there to guide or instruct participants?
-- Is there a way to easily incorporate the underlying data into a rectangular data format for statistical analysis or reporting?
\ No newline at end of file
+- Is there a way to easily incorporate the underlying data into a rectangular data format like CSV or an R `DataFrame` for statistical analysis or reporting?
```

Comment:
That's better.  Rectangular was a term we used in economics.  So it basically never translates to anything ever.  I will submit a bug for a separate pull request.